sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/metadata.yaml

```diff
@@ -15,7 +15,6 @@ labels:
 scheduling:
   dag_name: bqetl_analytics_aggregations
   task_name: active_users_aggregates_attribution_v1
-  date_partition_offset: -1
```

Comment:
These `bqetl_analytics_aggregations` tasks still have the same `date_partition_offset` (`/dags/bqetl_analytics_aggregations.py` isn't modified), but that setting is now determined automatically based on the recursive upstream dependencies.